app/serializers/trail_show_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class TrailShowSerializer < ActiveModel::Serializer
+  attributes :id, :name, :street, :city, :state, :zip, :start_latitude, :start_longitude, :length, :difficulty, :description, :ascent, :photo, :low_elevation, :high_elevation
+
+  has_many :reviews
+
+  def reviews
+    object.reviews
+  end
+end
```

Comment:
same as the other Trail serializer. Redundant

Reply:
Deleted method